doc/sailfish-webview-webview.qdoc

```diff
@@ -14,4 +14,20 @@
   import namespace.
 */
 
+/*!
+  \qmlproperty PopupProvider WebView::popupProvider
```

Comment:
Would be cool if this would provide a link to the PopupProvider as well.

Reply:
I'm not sure why, but it doesn't generate a link automatically.  Maybe it's a namespace issue?  But, I don't think it matters much, as I've added link in text and in \sa now also